### ISSUE TITLE
Version number would be nice

### ISSUE THREAD

**Issue by ondohotola** [YS-L/csvlens#21]
I tried `-v -V --version` but it doesn't seem to have a version number.

Maybe you want to add that?

**Comment by YS-L** [comment]
Good idea, will add that :+1:

**Comment by ondohotola** [comment]
```
cargo install csvlens
    Updating crates.io index
     Ignored package `csvlens v0.1.11` is already installed, use --force to override
```

All of `csvlens -v | -V --version | --Version` 

show errors like

`error: Found argument '-v' which wasn't expected, or isn't valid in this context`

**Comment by YS-L** [comment]
The new option is available in the latest main branch but not yet published to crates.io. I plan to do that soon and it will be available as part of version 0.1.12.

The expected format for the option is: 
```
  -V, --version                    Print version
```
